static/js/src/advantage/subscribe/react/components/Form/Feature/Feature.tsx

```diff
@@ -25,14 +25,35 @@ const Feature = () => {
 
   const infraOnlyDisabled = ProductTypes.desktop === productType;
 
+  const isDisabled = (support?: string) => {
```

Comment:
Same comments that I left for the other component, use `SupportEnum` and make the function argument required.